types/bull-arena/index.d.ts

```diff
@@ -33,7 +35,7 @@ declare namespace BullArena {
     interface QueueOptions {
         name: string;
         hostId?: string;
-        type?: "bull" | "bee";
+        type?: "bull" | "bee" | string;
```

Comment:
I am not sure how Arena deals with bullmq. Do we send "bullmq" as the type param when using bullmq? If we can make the type stronger then `string` here that would be better.

Reply:
Thanks for pointing this. I'll add `bullmq` but there is a reason for adding string because of a compilation error.
`string` is in use in prefix param. Similarly I used it here and it fixed the issue.

```
Type '{ name: string; hostId: string; type: string; url: string; }[]' is not assignable to type '((QueueOptions & PortHostConnectionOptions) | (QueueOptions & RedisUrlConnectionOptions) | (QueueOptions & RedisClientConnectionOptions))[]'.
  Type '{ name: string; hostId: string; type: string; url: string; }' is not assignable to type '(QueueOptions & PortHostConnectionOptions) | (QueueOptions & RedisUrlConnectionOptions) | (QueueOptions & RedisClientConnectionOptions)'.
    Type '{ name: string; hostId: string; type: string; url: string; }' is not assignable to type 'QueueOptions & RedisUrlConnectionOptions'.
      Type '{ name: string; hostId: string; type: string; url: string; }' is not assignable to type 'QueueOptions'.
        Types of property 'type' are incompatible.
          Type 'string' is not assignable to type '"bull" | "bee" | undefined'.
```

Reply:
I don't understand the compilation error. Does the compiler pass before your changes? Before your PR, `| string` didn't exist. That's why I don't think we need it now.

Reply:
Whenever I pass `type` parameter in Arena constructor, then this compilation error pops up.

Reply:
Could you add some tests to the definitions that show Arena construction?

Reply:
This compilation error was coming even before my changes. This error is not related to these changes.

Reply:
I have tested it by making a sample project. See `prefix` parameter in Arena constructor where it is allowed to have any string.